support/updateStylelintCustomSassFunctions.ts

```diff
@@ -15,7 +15,7 @@ function collectSassFiles(dir: string): string[] {
 
   try {
     fs.readdirSync(dir, { recursive: true, withFileTypes: true }).forEach(dirent => {
```

Comment:
Do we need to recursively check files starting in the root directory or can we be more specific?

The only relevant directories are `packages/calcite-components/src` and `packages/calcite-design-system/dist/scss` right?

Reply:
Is the suggestion to narrow the search scope mainly for performance reasons? I’d like to avoid an allowlist, so we don’t need to (remember to) update this script if our Sass file directories change.